Greenpill-class Scrying Vessel

BBBBAface opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

![2022 09 15-16 36 55](https://user-images.githubusercontent.com/19318747/190506173-921cab1d-f0c2-4c98-a26a-c119dc997be4.png)

![2022 09 15-16 36 49](https://user-images.githubusercontent.com/19318747/190506167-91a73e17-200a-4ebe-8205-4722fb9a47b1.png)

Adds a new ship, the Greenpill-class scrying vessel, meant for 2 people. This is the first of a series of wizard ships I am making, so its the smallest and worst equipt. This pr includes a few jobs as well.

Features:
Three crew maximum
Single-use spellbooks
No generators, the ship gets power from RT- I mean, Mana field condensers
Magic rake
Grimoire of Rock Revelation
Monkey



## Why It's Good For The Game

I feel like the current wizard ship, the Lamia, is underutilized due to being shit and overpriced. I also feel like it needs companion wizard ships. This PR is the first of a few that are aimed to make the Wizard Federation a little more represented and add some high-priced ships that are actually worth buying.

## Changelog
:cl:
add: Greenpill
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

BBBBAface committed:
Added whole complete project

shiptest didn't want it so

BBBBAface committed:
fixes errors

if not idk, feel free to fix them if you want to use the ship because i don't play voidcrew and apparently the codebase is different enough for many things to not work / be here

BBBBAface committed:
i lied theyre all fixed now

BBBBAface committed:
fr

BBBBAface committed:
frfr

BBBBAface committed:
fixes prefix

BBBBAface committed:
deletes grilles and swaps holopad

airlocks work good (thumbs up emoji)

BBBBAface committed:
fixes door tag

all good now

BBBBAface committed:
made price reasonable

BBBBAface committed:
Merge branch 'voidcrew:master' into master

BBBBAface committed:
reorganizes bridge and slightly un-nerfs sec

I looked at other ships in this codebase and realized the near complete lack of security gear would likely discourage people from picking this ship. Hopefully it's good in it's current state.

BBBBAface committed:
Merge branch 'voidcrew:master' into master

BBBBAface committed:
this is the file i shouldve uploaded

BBBBAface committed:
Tweaks to floors

Also adds turbine boards (@Bread Man#7984's idea)

BBBBAface committed:
Delete nanotrasen_tanker.json

BBBBAface committed:
Delete nanotrasen_tanker.dmm

BBBBAface committed:
changes NEU to NT-C

i just made it neu without looking at what it stood for lmao

BBBBAface committed:
minor tweaks

BBBBAface committed:
Delete NEU_tanker.json

BBBBAface committed:
changes output on tanks to manual valve

Making the tank connectors instead of open-ended manual valves is something the shiptest devs wanted me to change, even though existing ships have open-ended valves already. Tank connectors on the end of the pipe makes no sense whatsoever and i forgot to change it back. This one really irritated me, no clue why they would make me wait for 2 days telling me to change that and then say they didn't even want the ship

BBBBAface committed:
Merge branch 'voidcrew:master' into Greenpill

BBBBAface committed:
Delete NT-C_tanker.json

BBBBAface committed:
Delete nanotrasen_tanker.dmm

BBBBAface committed:
ignore previous commits, i am bad at github

I put my first PR, the nanotrasen tanker, on the master branch of my fork so now when i make new branches they have the commits already on them aaaaaaaaa

BBBBAface committed:
linter aaaaaa

BBBBAface committed:
why

BBBBAface committed:
Makes it KOS

BBBBAface committed:
adds autolathe and tweaks

BBBBAface committed:
Delete NEU_greenpill.json

BBBBAface committed:
linter aaaaaaa

BBBBAface committed:
last one

BBBBAface committed:
somehow my local version of this file is hated by linters

GuiltyNeko reviewed (commented):
It looks good at a cursory glance, are you sure you want ALL your slots to only be officer slots? That will prevent respawning, and give the ship a hardcap of 2 people and no more. Also, you have an assistant wizard outfit datum that will merge conflict (on top of the others) if any of your other PRs gets merged, and the assistant outfit datum isn't being used anyways.

BBBBAface committed:
Delete assistant.dm

dont need it, dont know why it was uploaded with this tbh

BBBBAface committed:
apprentice not officer anymore

BBBBAface committed:
tweaks job

BBBBAface committed:
tweaks gun sprite to be more wand-y

BBBBAface commented:
> It looks good at a cursory glance, are you sure you want ALL your slots to only be officer slots? That will prevent respawning, and give the ship a hardcap of 2 people and no more. Also, you have an assistant wizard outfit datum that will merge conflict (on top of the others) if any of your other PRs gets merged, and the assistant outfit datum isn't being used anyways.

added a third non-officer job for if people want to really cram in here, for some reason.

BBBBAface committed:
consolidating jobs onto this pr

to prevent future issues or whatever idk, the other wizard ships now rely on this

BBBBAface committed:
LINTERS

BBBBAface committed:
tweaks config

BBBBAface committed:
job tweaks

BBBBAface committed:
fixes pipes

GuiltyNeko reviewed (approved):
Looks good.
